templates/base/head_navbar.tmpl

```diff
@@ -22,13 +21,8 @@
 		{{if .ShowMilestonesDashboardPage}}<a class="item {{if .PageIsMilestonesDashboard}}active{{end}}" href="{{AppSubUrl}}/milestones">{{.i18n.Tr "milestones"}}</a>{{end}}
 		{{end}}
 		<a class="item {{if .PageIsExplore}}active{{end}}" href="{{AppSubUrl}}/explore/repos">{{.i18n.Tr "explore"}}</a>
-	{{else if .IsLandingPageHome}}
```

Comment:
Why we need to change these parts?

Reply:
@axifive suggested it and it made sense to generalize it.

Reply:
> @axifive suggested it and it made sense to generalize it.

I just suggested bring your solution to work correctly. Maybe need leave a separate link for the organization
https://github.com/go-gitea/gitea/pull/16844/files#r719655772